frontend/test/__support__/e2e/helpers/e2e-boolean-helpers.js

```diff
@@ -0,0 +1,24 @@
+// until we have a test dataset that includes boolean data, we can use this questions to test booleans
```

Comment:
@nemanjaglumac , would appreciate your eyes on this. I found this method of testing boolean filters in `filters.cy.spec.js`, and extracted it so I could use it again. 

Does this fit with the patterns of other e2e helpers?
Can you think of a better way to test booleans without boolean data in the sample DB?

Reply:
Seems like a very good solution. Thanks for doing this.

Left some inline comments that need to be addressed, though.